lib/SearchAndSort/SearchAndSort.js

```diff
@@ -250,9 +250,13 @@ class SearchAndSort extends React.Component {
       stripes,
     } = this.props;
 
+    // Use local storage to remember whether the filter pane has been toggled off or not.
+    this.filterPaneVisibilityKey = '@folio/inventory/inventoryFilterPaneVisibility';
```

Comment:
@Baroquem I think this will have to be different for each ui module right?
I wonder if we could use a `withNamespace` HOC from stripes-core: 

https://github.com/folio-org/stripes-core/blob/0fcafe70b8a25525ef488ab6e7e800c148799e4d/src/components/Namespace/withNamespace.js#L9 

to get a unique key for each module.

You could try to wrap the entire `SearchAndSort` with `withNamespace` and you should get a `namespace` prop as a string.

Reply:
Yes, of course you're both right -- this shouldn't be hard-coded here. Thanks for the `withNamepsace` tip, Michal; that's exactly what I needed!